### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by docgonzo2015** [syagent/agent-2#1] [Ideas]
Hi!
I could translate it to polish and german, if u want to

**Comment by Rashe** [comment]
Hey, thanks for suggestion. It's cool idea to make this project multi language. 

Current I'm working on heartbeat monitoring. But I would like to add languages in future.

**Reply by docgonzo2015** [reply]
Then you can already insert the Int. with new code :-)

A good example : https://www.gnu.org/software/gettext/manual/html_node/Preparing-Shell-Scripts.html
